### PR TITLE
Introduce text in SupportView and RequestView

### PR DESCRIPTION
**Pull request by martinfrancois**
Introduce text according to mockups, not yet with strings class
Also some of the styling on RequestView has been done
Also did some refactoring

### CODE REVIEW

File: src/ch/imedias/rsccfx/view/RsccRequestView.java [hunk 1]
```diff
@@ -29,25 +28,26 @@
...
-  final TitledPane supporterAdminPane = new TitledPane();
-
-  final Text descriptionTxt = new Text();
-  final Text additionalDescriptionTxt = new Text();
+  final TitledPane predefinedAddressesPane = new TitledPane();
 
   final TextField generatedKeyFld = new TextField();
 
   final Button reloadKeyBtn = new Button();
+  final Button readyBtn = new Button();
```
Comment (rmosimann) [inline src/ch/imedias/rsccfx/view/RsccRequestView.java:49]: What functionality will this Button have?

**Review by rmosimann — APPROVED** [review]
This looks good to me, but i think we have to discuss that one button in the team anyway (is it needed?)

**Comment by rmosimann** [comment]
Feel free to merge, but maybe you also want to have a feedback from the GUI people  (@l-marchesi @JenniferMue )  who are more involved in those topics...

File: src/ch/imedias/rsccfx/view/RsccRequestView.java (hunk 1 above)
Comment (l-marchesi) [reply src/ch/imedias/rsccfx/view/RsccRequestView.java:49]: That's a really good question - as far as I understood from @pwigger it should show if it's possible to connect or not. However, we could realise this with a label or so.

Maybe we should bring it up on the next team meeting

**Comment by l-marchesi** [comment]
We were involved in the process, we asked him to do this, since we were working on the presentation. So ready to merge from my side :)

**Pull request merged**